PROTO_tests/tests/numeric_test.py

```diff
@@ -455,6 +507,8 @@ def test_value_counts(self, num_type):
 
     def test_value_counts_error(self):
         pda = ak.linspace(1, 10, 10)
+        # with pytest.raises(TypeError):
+        #   ak.value_counts(pda)
```

Comment:
Why is this commented out?

Reply:
Because once again, I forgot to simply delete it after testing.  So that's fixed.  This used to raise a TypeError, but doesn't anymore.